test/fixtures/plugin.legend/legend-doughnut-right-center-mulitiline-labels.json

```diff
@@ -0,0 +1,27 @@
+{
+	"config": {
+		"type": "doughnut",
+		"data": {
+			"labels": ["", ["", "", "", "", ""], "", "", ""],
+			"datasets": [{
+				"data": [10, 20, 30, 40],
+				"backgroundColor": "#00ff00",
+				"borderWidth": 0
+			}]
+		},
+		"options": {
+			"plugins": {
+				"legend": {
+					"position": "right",
+					"align": "center"
+				}
+			}
+		}
+	},
+	"options": {
```

Comment:
I think it would be good to add the `spriteText` option so that we can see the multiline label (would require adding some text in the `labels`) as well